arch/arm/src/a1x/a1x_lowputc.c

```diff
@@ -27,13 +27,15 @@
 #include <stdint.h>
 
 #include <arch/irq.h>
-#include <arch/board/board.h>
 
 #include "arm_internal.h"
 #include "a1x_config.h"
-#include "hardware/a1x_uart.h"
 #include "a1x_pio.h"
 
+#include "hardware/a1x_uart.h"
+
+#include <arch/board/board.h>
```

Comment:
why not keep it in the original place?

Reply:
it looked more consistent, where u have nuttx specific imports first, then hardware includes then board.h. But i guess i could just revert it as it is unnecessary,

Reply:
normally, <> puts before "".

Reply:
ah alright, i'll revert them then